pkg/services/ngalert/store/alert_rule.go

```diff
@@ -665,3 +667,23 @@ func (st DBstore) validateAlertRule(alertRule ngmodels.AlertRule) error {
 
 	return nil
 }
+
+// ListNotificationSettings fetches all notification settings for given organization
+func (st DBstore) ListNotificationSettings(ctx context.Context, orgID int64) (map[ngmodels.AlertRuleKey][]ngmodels.NotificationSettings, error) {
+	var rules []ngmodels.AlertRule
+	err := st.SQLStore.WithDbSession(ctx, func(sess *db.Session) error {
+		return sess.Table(ngmodels.AlertRule{}).Select("uid, notification_settings").Where("notification_settings IS NOT NULL AND org_id = ?", orgID).Find(&rules)
```

Comment:
TDB: While I was testing I found that the cell contains `null` value, i.e. actually it's not NULL. We need to adjust this query